server/routes/auth.routes.js

```diff
@@ -395,8 +428,8 @@ function convertCASNamesToSRT (cas_userinfo) {
   srt_userinfo['lastName'] = srt_userinfo['last-name']
   delete srt_userinfo['last-name']
 
-  srt_userinfo['agency'] = translateCASAgencyName(srt_userinfo['org-agency-name'])
-  delete srt_userinfo['org-agency-name']
+  srt_userinfo['agency'] = grabAgencyFromEmail(srt_userinfo['email'])
```

Comment:
Not sure this update is needed, since it seems like it is only used for Max login. But correct me if I'm wrong.